RIOT-OS/RIOT
# boards*: remove remaining uses of export with some openocd related variables

### Contribution description

This PR removes remaining use of global `export` for openocd related variables. It is similar to #13452 and #13480.

Build system static check is also adapted.




### Testing procedure

- A green CI/Murdock
- Verify that the adapted boards (nrf52 when softdevice is used, phynode-kw41z, mulle, seeduino_arch-pro, pba-xxx) can still be flashed with openocd
- Verify reset still work on hifive1  (who has that boards ?)




### Issues/PRs references

Related to #10850
